### ISSUE TITLE
Error: 0x80370102 The virtual machine could not be started because a required feature is not installed

### ISSUE THREAD

**Issue by ThePresence** [microsoft/WSL#8730]
### Version

Microsoft Windows Server 2022

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Repro Steps

I'm running Windows Server 2022 base with container on **AWS EC2 i3 instance**.

Installed WSL using command
`wsl --install`
after installation, I've restarted the system, and I'm getting this error `"Error: 0x80370102 The virtual machine could not be started because a required feature is not installed."`

I've also tried enabling hyper-v but still the same issue.

### Expected Behavior

Ubuntu installed successfully.

### Actual Behavior

"Error: 0x80370102 The virtual machine could not be started because a required feature is not installed."

**Comment by magicweirdoo** [comment]
Try `bcdedit /set hypervisorlaunchtype auto` in CMD and then reboot, that worked for me.

**Comment by ThePresence** [comment]
> Try `bcdedit /set hypervisorlaunchtype auto` in CMD and then reboot.

Hi,

Thank you for the response, but I'm still getting the same error after running that command.

**Comment by Tao-2019** [comment]
Hi, I am having the same problem with my project, can you share your solution if you found it? thanks in advance.

**Comment by hisbanliu** [comment]
Has this problem been fixed?

**Comment by Tao-2019** [comment]
> Has this problem been fixed?

Yes, it was fixed. Solution is to pick of different VM framework (With metal layer support), then WSL2 can be installed.

**Comment by hisbanliu** [comment]
> > Has this problem been fixed?
> 
> Yes, it was fixed. Solution is to pick different VM framework (With metal layer support) from AWS, then WSL2 can be installed.

What should be done about Microsoft azure VMs?
